src/tests/execute-wp-cli.test.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @jest-environment node
+ */
+
+import getWpNowConfig from '../../vendor/wp-now/src/config';
+import { downloadWPCLI } from '../../vendor/wp-now/src/download';
+import { executeWPCli } from '../../vendor/wp-now/src/execute-wp-cli';
+import getWpCliPath from '../../vendor/wp-now/src/get-wp-cli-path';
+import startWPNow from '../../vendor/wp-now/src/wp-now';
+
+jest.mock( '../../vendor/wp-now/src/download' );
+jest.mock( '../../vendor/wp-now/src/wp-now' );
+jest.mock( '../../vendor/wp-now/src/get-wp-cli-path' );
+jest.mock( '../../vendor/wp-now/src/config' );
+
+const mockPhpInstance = {
+	setSapiName: jest.fn().mockResolvedValue( undefined ),
```

Comment:
I used `undefined` but I am not sure if this is the best approach. The reason for this setup is to show that the function completed. These functions do not return meaningful values. Perhaps, something like below could be more clear?

```suggestion
jest.fn().mockResolvedValue('setSapiName resolved');
const mockMkdir = jest.fn().mockResolvedValue('mkdir resolved');
const mockWriteFile = jest.fn().mockResolvedValue('writeFile resolved');
const mockMount = jest.fn().mockResolvedValue('mount resolved');
```